src/LogUtil.ts

```diff
@@ -48,13 +54,15 @@ export function info(line: string): void {
 /**
  * @returns A boolean indicating whether a logger has been set within the AWS SDK.
  */
```

Comment:
do we need @internal here?

Reply:
Yes, because otherwise these are exported to the API